commands/secret_create.go

```diff
@@ -104,3 +110,21 @@ func readSecretFromFile(secretFile string) (string, error) {
 	fileData, err := ioutil.ReadFile(secretFile)
 	return string(fileData), err
 }
+
+// Kubernetes DNS-1123 Subdomain Regex
+// https://github.com/kubernetes/kubernetes/blob/6902f3112d98eb6bd0894886ff9cd3fbd03a7f79/staging/src/k8s.io/apimachinery/pkg/util/validation/validation.go#L131
+const (
+	dns1123LabelFmt          string = "[a-z0-9]([-a-z0-9]*[a-z0-9])?"
+	dns1123SubdomainFmt      string = dns1123LabelFmt + "(\\." + dns1123LabelFmt + ")*"
+	invalidSecretNameMessage string = `ERROR: invalid secret name: %s, secret name must consist of lower case alphanumeric characters, '-' or '.', and must start and end with an alphanumeric character (regex used for validation is %s)`
```

Comment:
I'd prefer to see this error message spread out one multiple lines. Having to scroll, or wrapped text makes it difficult to read.

Also, perhaps it would be better to _describe_ the regex rather than printing it out in the raw
Something like:
```
Error: Invalid secret name.
Secret name must start and end with alphanumeric character
and can only contain lower-case alphanumeric characters, '.', or '-'
```